jekyll/assets/js/highlightJSBadge.js

```diff
@@ -0,0 +1,41 @@
+window.onload = () => {
+   // https://app.optimizely.com/v2/projects/16812830475/experiments/20580130094/variations
+
+   window.OptimizelyClient.getVariationName({
+      featureName: 'add_copy_code_button_to_docs',
```

Comment:
Please prefix you experiment name with `dd_` 🙏

Reply:
quick note on this: make sure to update the experiment key name in optimizely as well after updating 🙏